zul/src/archive/web/js/zul/menu/Menubar.js

```diff
@@ -159,15 +160,15 @@ zul.menu.Menubar = zk.$extends(zul.Widget, {
 		var fixedSize = nodeWidth - zk(this.$n('left')).offsetWidth() - zk(this.$n('right')).offsetWidth();
 		if (this._scrolling) {
 			body.style.width = jq.px0(fixedSize);
-			this._fixScrollPos(node);
+			this._fixScrollPos(children.last()[0]);

```

Comment:
what if no children?

Reply:
`children.last()[0]` would be undefined.
example: `jq().last[0]`